src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/Utilities.cs

```diff
@@ -144,8 +144,8 @@ internal static object GetPrimitiveDefaultValue(Type type)
                     return default(float);
                 case TypeCode.Double:
                     return default(double);
-                //case TypeCode.DBNull:
-                //    return default(DBNull);
+                case TypeCode.DBNull:
+                    return default(DBNull);
```

Comment:
Yup that's coming in a separate PR. Actually pushing this PR to quickly see the CI pipeline turn green. I'll address a few of these, modulo the ones that you're dealing with wrt `BindingFlags` overloads etc.